source/leuven/mmlsquota.rst

```diff
@@ -0,0 +1,24 @@
+.. _mmlsquota:
+
+Total disk space on GPFS file systems
+=====================================
+
+On some parallel file systems, notably ``$VSC_SCRATCH`` on KU Leuven
+infrastructure (Tier-1 and Tier-2), ``mmlsquota`` should be used to
+display the total disk space you use and the limits you have.
```

Comment:
@gjbex why `myquota` cannot be used for for $VSC_SCRATCH on KU Leuven infra as well?

Reply:
replacing 'should' with 'can' is indeed better in that case.